### PR TITLE
[6.4.z] Nailgun version as per upgrade branch

### PR DESCRIPTION
**Pull request by jyejare**
Installation of nailgun as per satellite upgrade branch

### CODE REVIEW

**Comment by jyejare** [comment]
Reviewers, please use squash and merge to merge 2 commits in one.

File: setup.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
     packages=find_packages(),
     install_requires=[
         'Fabric3',
-        'fauxfactory==2.0.9',
+        'fauxfactory==3.0.2',
```
Comment (ntkathole) [inline setup.py]: robottelo 6.3.z branch needs fauxfactory==2.1.1 ?

File: upgrade/satellite.py [hunk 2]
```diff
@@ -168,12 +167,11 @@ def satellite6_upgrade():
         postup_time = datetime.now().replace(microsecond=0)
         logger.highlight('Time taken for Satellite Upgrade - {}'.format(
             str(postup_time-preup_time)))
-    set_hammer_config()
     # Rebooting the satellite for kernel update if any
     reboot(180)
     host_ssh_availability_check(env.get('satellite_host'))
     # Test the Upgrade is successful
-    hammer('ping')
+    run('hammer ping', warn_only=True)
```
Comment (ntkathole) [inline upgrade/satellite.py]: Non-blocking comment. No need of warn_only here.

**Comment by san7ket** [comment]
Two  different commits here, there a different PR raised for the 2nd commit

File: setup.py (hunk 1 above)
Comment (jyejare) [reply setup.py]: This Pr is for 6.4.z branch.

File: upgrade/satellite.py (hunk 2 above)
Comment (jyejare) [reply upgrade/satellite.py]: resolving it in appropriate PR's. I squashed my PR and has only installation related changes.

**Pull request merged**